cmd/api/src/test/integration/harnesses.go

```diff
@@ -1251,7 +1252,77 @@ func (s *ADCSESC1Harness) Setup(graphTestContext *GraphTestContext) {
 	graphTestContext.NewRelationship(s.CertTemplate44, s.EnterpriseCA4, ad.PublishedTo)
 	graphTestContext.NewRelationship(s.CertTemplate45, s.EnterpriseCA4, ad.PublishedTo)
 	graphTestContext.NewRelationship(s.CertTemplate46, s.EnterpriseCA4, ad.PublishedTo)
+}
 
+type EnrollOnBehalfOfHarnessTwo struct {
+	Domain2        *graph.Node
+	AuthStore2     *graph.Node
+	RootCA2        *graph.Node
+	EnterpriseCA2  *graph.Node
+	CertTemplate21 *graph.Node
+	CertTemplate22 *graph.Node
+	CertTemplate23 *graph.Node
+	CertTemplate24 *graph.Node
+	CertTemplate25 *graph.Node
+}
+
+func (s *EnrollOnBehalfOfHarnessTwo) Setup(gt *GraphTestContext) {
+	sid := RandomDomainSID()
+	certRequestAgentEKU := make([]string, 0)
+	certRequestAgentEKU = append(certRequestAgentEKU, adAnalysis.EkuCertRequestAgent)
+	emptyAppPolicies := make([]string, 0)
+	sid = RandomDomainSID()
```

Comment:
`sid` is declared and immediately reassigned without using the first assignment value so line 1274 can be axed.